packages/react-instantsearch-hooks-dom/src/ui/PoweredBy.tsx

```diff
@@ -1,21 +1,57 @@
 import React from 'react';
-import { usePoweredBy } from 'react-instantsearch-hooks';
 
-export function PoweredBy() {
-  const { url } = usePoweredBy();
+import { cx } from './lib/cx';
 
+export type PoweredByClassNames = {
+  /**
+   * Class names to apply to the root element
+   */
+  root: string;
+  /**
+   * Class names to apply to the link element
+   */
+  link: string;
+  /**
+   * Class names to apply to the SVG logo
+   */
+  logo: string;
+};
+
+export type PoweredByProps = React.HTMLAttributes<HTMLDivElement> & {
+  classNames: Partial<PoweredByClassNames>;
```

Comment:
if we follow the same pattern as highlight, in the ui component the class names are required, and passed directly from the prop; in the widget they are optional and get the default value.

Reply:
See my [explanation](https://github.com/algolia/react-instantsearch/pull/3391#issue-1178092284) for why.